components/OpportunityCard.jsx

```diff
@@ -0,0 +1,168 @@
+import styled from "styled-components";
+import { Text } from "./Text";
+import { media } from "../utils";
+
+export const OpportunityCard = ({
+  name,
+  location,
+  category,
+  organization,
+  postingDate,
+  logo,
+	link,
+  ...props
+}) => {
+  return (
+    <Wrapper {...props}>
+      <Logo src={logo} />
+			<RightContainer>
+				<InfoContainer>
+					<div>
+						<Name>{name}</Name>
+						<Organization>{organization}</Organization>
+					</div>
+					<div>
+						<Category>{category}</Category>
+						<BottomText>
+							<Location>{location}</Location>
+							<PostingDate>Posted {postingDate}</PostingDate>
+						</BottomText>
+					</div>
+				</InfoContainer>
+			</RightContainer>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div`
+  ${({ theme }) => `
+		border-color: ${theme.font.text};
+		box-shadow: ${theme.boxShadow.topBottom};
+	`};
+  border-radius: 20px;
+  display: flex;
+  flex-direction: row;
+	width: calc(100%-40px);
+  height: 190px;
+  padding: 0;
+  margin: 10px;
```

Comment:
The margin isn't showing up at smaller widths (card doesn't grow as the content gets closer to the edge), I think because the height is forcing the card to only be so tall.